xesmf/backend.py

```diff
@@ -58,7 +58,13 @@ def warn_lat_range(lat):
 
 class Grid(ESMF.Grid):
     @classmethod
-    def from_xarray(cls, lon, lat, periodic=False, mask=None):
+    def from_xarray(
+        cls,
+        lon: np.ndarray[float, int],
+        lat: np.ndarray[float, int],
+        periodic: bool = False,
+        mask=None,
```

Comment:
```suggestion
        mask: Optional[np.ndarray[float, int]] = None,
```
Suggested for consistency with other args of `from_xarray`